### ISSUE TITLE
Add +++ for escape and auto-baud rate

### ISSUE THREAD

**Issue by samighi** [esp8266/at-command-set#1]
We should add +++ to reset the chip and auto select baud rate

**Comment by igrr** [comment]
Any particular reason why +++?
It's normally an escape sequence used to return from data mode to command mode.
IIRC, AT+IPR=0 triggers baud rate detection.
There's an UART_AUTOBAUD register in ESP8266, I will poke around and see how it works.

**Comment by mstempin** [comment]
I agree with @igrr, the +++ sequence should be reserved for its standard purpose.

**Comment by samighi** [comment]
that is fine (RE: +++). Do we have (tested) an auto baud selector? type it in and chip will match speed? ﻿

**Comment by igrr** [comment]
Not sure how that would work.
On ESP8266 you have to explicitly enable autobaud mode. Having a command that would enable autobaud mode looks a bit pointless — if your baud rate doesn't match the baud rate set on the chip's uart, the command will not be interpreted correctly. And if the rates match, there's no reason to enable autobaud mode.
So the only use case would be to turn autobaud mode on firmware start. But we don't need a command for that.
